modular_skyrat/modules/admin/code/fix_chat.dm

```diff
@@ -0,0 +1,7 @@
+/client/proc/fix_say()
+    set name = "FIX SAY"
+    set category = "Admin"
+    set desc = "fixes bug where people can't say shid"
+    for(var/player in GLOB.player_list)
```

Comment:
this list is supposed to be cleaning itself of nulls, i suspect you have a runtime causing this problem

Reply:
Apparently it happens on tg and bee as well

Reply:
This isnt a problem on Beestation or Shiptest